### DISCUSSION TITLE
Slack

### DISCUSSION THREAD

**Discussion by ups7** [ghostfolio/ghostfolio#2033] [Q&A] [answered]
How do you join the Slack channel for Ghostfolio? It seems you need to be invited to it first, and I can't see a way to do that. Sorry if I'm missing something obvious!

Thanks in advance!

**Comment by dtslvr** [comment, marked as answer]
Hi @ups7

Can you try this: [Join Ghostfolio on Slack](https://join.slack.com/t/ghostfolio/shared_invite/zt-vsaan64h-F_I0fEo5M0P88lP9ibCxFg)

**Reply by ups7** [reply]
Thank you, that worked!
